build_all.sh

```diff
@@ -48,6 +49,7 @@ make -j 8 VERBOSE=1
 make install
 
```

Comment:
I always used 'make test'. Using 'ctest' is better. Please remove the 'make test' line.

Reply:
Fixed at 997201d.